javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,19 @@
+var fullName = 'Nicholas Fernandez';
+var isCool = 'false';
+var totalPets = 1;
+
+console.log('value of fullName', fullName);
+console.log('value of isCool', isCool);
+console.log('value of totalPets', totalPets);
+
+console.log('typeof fullName', typeof fullName);
+console.log('typeof isCool', typeof false);
```

Comment:
you should be using the variable here. All the console logs should use the variables.